port/zos390/omrvmem.c

```diff
@@ -845,6 +857,14 @@ reserve_memory_with_moservices(struct OMRPortLibrary *portLibrary, struct J9Port
 		Trc_PRT_vmem_reserve_memory_using_moservices_invalid_page_size(params->pageSize);
 		rc = -1;
 	}
+
+	/* Explicitly set the storage flag for executable. If left unspecified, the default is the storage will be executable,
+	 * unless DIAGxx IARNONEXEC64 trap is set. The __MOPL_EXECUTABLE flag is ignored on systems that do not support the flag.
+	 */
+	if (OMR_ARE_ANY_BITS_SET(OMRPORT_VMEM_MEMORY_MODE_EXECUTE, params->mode)) {
```

Comment:
The order of value and bits should be swapped as per the macro's definition:
https://github.com/eclipse/omr/blob/26e52210d25d5b3cd84b1573d4060ef7b663303e/include_core/omrcomp.h#L577
```suggestion
	if (OMR_ARE_ANY_BITS_SET(params->mode, OMRPORT_VMEM_MEMORY_MODE_EXECUTE)) {
```